api/libproject/devstart.rb

```diff
@@ -200,52 +206,53 @@ def run_cloud_sql_proxy(project, creds_file)
   return pid
 end
 
+# Common.run_inline uses spawn() which doesn't handle pipes/redirects.
```

Comment:
Note that there is `common.pipe` for this purpose: https://github.com/dmohs/project-management/blob/master/common.rb#L144